src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -405,6 +412,34 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
     }
   }
 
+  ngAfterViewInit() {
+    if (! flags.friendlyCaptchaEnabled) {
+      return;
+    }
+
+    if (! isPlatformBrowser(this.platformId)) {
+      return;
+    }
+
+    const friendlyCaptcha = this.friendlyCaptcha;
+
+    if (! friendlyCaptcha) {
+      // should always be loaded from the template, but not present in tests.
+      return;
+    }
+    const widget = new WidgetInstance(friendlyCaptcha.nativeElement, {
+      doneCallback: (solution) => {
+        this.idCaptchaCode = solution;
+        console.log('DONE: ', solution);
+      },
+      errorCallback: (b) => {
+        console.log('FAILED', b);
+      },
```

Comment:
noticed these console logs I pasted in from the friendly captcha docs. Think I'll delete. Not sure if/when we should expect an error, I guess we should add an apologetic error message under the captcha in case of error.